### ISSUE TITLE
Error starting with Emacs 27 package-quickstart

### ISSUE THREAD

**Issue by drot** [joaotavora/sly#216]
When enabling the new `package-quickstart` feature SLY fails to load with the following message:

```
Debugger entered--Lisp error: (file-missing "Cannot open load file" "No such file or directory" "sly")
  #<subr autoload-do-load>((autoload "sly" "Start a Lisp implementation and connect to it.\n\n  ..." t nil) sly nil)
  apply(#<subr autoload-do-load> ((autoload "sly" "Start a Lisp implementation and connect to it.\n\n  ..." t nil) sly))
  autoload-do-load((autoload "sly" "Start a Lisp implementation and connect to it.\n\n  ..." t nil) sly)
  command-execute(sly)
```

Disabling this feature allows SLY to start normally again.
Tested on `GNU Emacs 27.0.50 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.24.5) of 2019-02-10`

**Comment by joaotavora** [comment]
Dang, package-quickstart is news to me. ..

How did you "enable" package-quickstart?  Do you enable it for a specific package or for all packages at once?

**Comment by drot** [comment]
Just with `(setq package-quickstart t)` in my init file and M-x `package-quickstart-refresh` to create the `~/.emacs.d/package-quickstart.el` file.

**Comment by joaotavora** [comment]
Not sure if this isn't a `package.el` bug...  Perhaps you should `M-x report-emacs-bug` or hail someone who knows how it works (@monnier?)

**Comment by drot** [comment]
Yeah, same issue with SLIME as well.

**Comment by joaotavora** [comment]
I wouldn't be suprised if SLY/SLIME were both my fault tho :-) as I did the package.el packaging for both (I think)

**Comment by drot** [comment]
Reported as [#34413](https://debbugs.gnu.org/cgi/bugreport.cgi?bug=34413).

**Comment by joaotavora** [comment]
Solved by #244, according to @drot
